src/tests/Drowpdown/Dropdown.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { render, screen } from "@testing-library/react";
+import { act } from "react-dom/test-utils";
+import Dropdown from "../../components/Dropdown/Dropdown";
+
+test("renders Dropdown component correctly", () => {
+  render(<Dropdown />);
+  const dropdownElement = screen.getByTestId("dropdown");
+  expect(dropdownElement).toBeInTheDocument();
+});
+
+test("all values are in dropdown", () => {
+  render(<Dropdown />);
+  const dropdownElement = screen.getByTestId("term-list");
+  expect(dropdownElement).toHaveTextContent("Net 1 day");
```

Comment:
Podes usar Regex, entonces si cambia alguna letra, no está atado a la integración.

```suggestion
  expect(dropdownElement).toHaveTextContent(/Net 1 day/i);
```

Reply:
Corregido!!